QCflow/torsion_run.py

```diff
@@ -5,27 +5,19 @@
 from QCflow.torsion_parser import *
 
 
-def run_torsion(fragment_smi_file, functional='B3LYP', basis_set='6-31G*'):
+def run_torsion(mol_dic, functional='B3LYP', basis_set='6-31G*'):
 
     '''
-    When provided with a .smi file with all the fragments, this function
-    combines all the fragemnts into dimers and then creates the SLURM and
-    guassian input file and then submits the jobs
+    When provided with a .json file of all dimers, this function
+    creates the SLURM and guassian input file and then submits the jobs
 
-    fragment_smi_file : name of the .smi file containing the fragments
+    mol_dic : name of the .json file containing the dimers
 
     functional : Preset is B3LYP
 
     basis_set : Preset is 6-31G*
     '''
 
-    #Loads the smi file and creates a dictionary of fragments
-    frag_dic = read_fragments(fragment_smi_file)
-    #Creates a dictionary of all the dimers from the fragment dictionary
-    mol_dic = make_dimer_dic(frag_dic)
-    #saves the dictionary as a json file
-    save_dictionary(mol_dic, 'mol_dic.json')
-
```

Comment:
Removes making the dimer. I found this was specific user cases rather than general so removed this and users should combine before this stage